nvflare/app_common/widgets/streaming.py

```diff
@@ -202,30 +205,6 @@ def _log(self, tag: LogMessageTag, msg: str, event_type: str, *args, **kwargs):
         with self.engine.new_context() as fl_ctx:
             send_analytic_dxo(self, dxo=dxo, fl_ctx=fl_ctx, event_type=event_type)
 
-    def info(self, msg: str, *args, **kwargs):
```

Comment:
Since these methods are deleted, we should no longer need the _log() method.

Reply:
Moved the streaming events to AppEventType.